include/seqan3/range/views/enforce_random_access.hpp

```diff
@@ -114,7 +105,7 @@ class view_enforce_random_access : public std::ranges::view_interface<view_enfor
     constexpr auto end() noexcept
     {
         if constexpr (std::ranges::common_range<urng_t>)
-            return enforced_random_access_iterator<decltype(std::ranges::end(urng))>{std::ranges::end(urng)};
+            return enforced_random_access_iterator<decltype(std::ranges::cend(urng))>{std::ranges::cend(urng)};
```

Comment:
I think this is wrong, because this is not the const qualified `end()`, the next `end()` is const and alrady uses `std::ranges::cend`.

Reply:
You are right. I'm sorry, fixed.